frontend/src/components/pages/SearchReviews/SearchReviews.tsx

```diff
@@ -78,27 +79,33 @@ const SearchReviews = (): React.ReactElement => {
       newSearchUrl.toString(),
     );
 
-    // todo call backend search filtering endpoint to retrieve reviews
-    setDisplayedReviews(mockReviews);
-
-    setLoading(false);
+    reviewAPIClient
+      .getReviews(searchText, 25, 0)
+      .then((reviewResponse: PaginatedReviewResponse) => {
+        setDisplayedReviews(mapReviewResponseToReview(reviewResponse.reviews));
+        setLoading(false);
+      });
   }, [searchText, genresFilter, ageRangeFilter]);
 
   if (loading) {
     return <LoadingSpinner mt="21%" />;
```

Comment:
I think it looks better if you only cover the grid with the loading spinner, not the whole page (ie leave the search bar)